EOLib/PacketHandlers/Guild/GuildAgreeHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using AutomaticTypeMapper;
+using EOLib.Domain.Character;
+using EOLib.Domain.Login;
+using EOLib.Domain.Notifiers;
+using EOLib.Net.Handlers;
+using Moffat.EndlessOnline.SDK.Protocol.Net;
+using Moffat.EndlessOnline.SDK.Protocol.Net.Server;
+
+namespace EOLib.PacketHandlers.Guild
+{
+    [AutoMappedType]
+
+    public class GuildAgreeHandler : InGameOnlyPacketHandler<GuildAgreeServerPacket>
+    {
+        private const byte JoinGuildSfx = 18;
```

Comment:
Can we use [SoundEffectID JoinParty](https://github.com/ethanmoffat/EndlessClient/blob/master/EndlessClient/Audio/SoundEffectID.cs) instead of magic numbers :) ?

Reply:
You can make it JoinGuild = JoinParty = TradeAccepted

Reply:
Problem is the sound effect enum isn't available in this project. The shop packet handler is the same

Reply:
Going to make this change as a separate PR once guilds are completed